app/components/Tab/Tab.jsx

```diff
@@ -75,14 +72,14 @@ export default class Tab extends Component
 
     reloadIfActive()
     {
-        const { isActiveTab } = this.props;
-
+        const { isActiveTab, pageLoaded } = this.props;
         if ( !isActiveTab )
         {
             return;
         }
 
         this.reload();
+        pageLoaded();
```

Comment:
This should ideally only be triggered by the webview event. See: https://github.com/bochaco/peruse/blob/66dc831b65ceec9944e0b0858379d0fd6ebf2659/app/components/Tab/Tab.jsx#L109